nchelpers/__init__.py

```diff
@@ -504,6 +494,27 @@ def getdottedattr(obj, dotted_attr):
             except:
                 raise missing_attribute(attr)
 
+        def __get_metadata_standard__(self):
```

Comment:
Eh, tiny comment here: I think "dunder" identifiers are (mainly) for magic methods like `__str__`, `__eq__`, etc. that the language uses to implement built-in functions and operators such as `str()`, `==`, etc. So maybe this should just be  `_get_metadata_standard`?